.github/workflows/test.yaml

```diff
@@ -346,3 +351,10 @@ jobs:
           KWOK_MODE: StableFeatureGateAndAPI
         run: |
           ./hack/e2e-test.sh kwokctl/kwokctl_${{ matrix.kwokctl-runtime }}
+
+      - name: Upload logs
+        uses: actions/upload-artifact@0b7f8abb1508181956e8e162db84b466c27e18ce # v3.1.2
+        if: failure()
+        with:
+          name: kwok-logs-${{ github.run_id }}-${{ matrix.os }}-${{ matrix.kwokctl-runtime }}
```

Comment:
What if different versions of nerdctl fail at the same time?

Reply:
https://github.com/actions/upload-artifact#uploading-to-the-same-artifact

Maybe we should add version of nerdctl to the name of the artifact ？